examples/README.md

```diff
@@ -120,6 +120,54 @@ $ PYTHONPATH=../ ../krun.py example.krun
 You should see a log scroll past, and results will be stored in the file:
 `../krun/examples/example_results.json.bz2`.
 
+## Using a Krun results file
+
+Krun generates a bzipped Json file containing results of all executions.
+The structure of the Json results is as follows:
+
+```python
+{
+    'audit': '',  # A unicode object containing platform information
+    'config': '', # A unicode object containing your Krun configuration
+    'data': {     # A dict object containing timing results
+        u'bmark:VM:variant': [  # A list of lists of timing results
+            [ ... ], ...        # One list per execution
+        ]
+  }
+}
+```
+
+Often it is useful to check the audit or configuration that a result
+file was generated with.
+To do this, call Krun with the `--dump-audit` or `--dump-config` options:
+
+```bash
+$ python krun.py --dump-config examples/example_results.json.bz2
+INFO:root:Krun starting...
+[2015-11-02 14:23:31: INFO] Krun starting...
+import os
+from krun.vm_defs import (PythonVMDef, JavaVMDef)
+from krun import EntryPoint
+
+# Who to mail
+MAIL_TO = []
+...
+
+$ python krun.py --dump-audit examples/example_results.json.bz2
+{u'cpuinfo': u'processor\t: 0\nvendor_id\t: GenuineIntel\ncpu family\t:
```

Comment:
I think it would be a surprise to the user if it wasn't. If you want to make it otherwise, I should add some clarity to the documentation to explain...